js/gallery.js

```diff
@@ -0,0 +1,62 @@
+const _galleryContent = `
+<div id="views">
+	<div class="selected">
+		<i class="material-icons">grid_on</i>
+	</div>
+	<div>
+	<i class="material-icons outlined darkable-icon">view_day</i>
+	</div>
+	<div>
+		<i class="material-icons outlined darkable-icon">movie</i>
+	</div>
+	<div>
+		<i class="material-icons grey-text">live_tv</i>
+	</div>
+	<div>
+		<i class="material-icons outlined darkable-icon">person_pin</i>
+	</div>
+</div>
+<div class="gallery">
+  <div class="img-container">
+    <img src="images/12668.jpg" width="100%" height="100%"/>
+  </div>
+  <div class="img-container">
+    <img src="images/30291.jpg" width="100%" height="100%"/>
+  </div>
+  <div class="img-container">
+    <img src="images/308412.jpg" width="100%" height="100%"/>
+  </div>
+  <div class="img-container">
+    <img src="images/34104.jpg" width="100%" height="100%"/>
+  </div>
+  <div class="img-container">
+    <img src="images/347587.jpg" width="100%" height="100%"/>
+  </div>
+  <div class="img-container">
+    <img src="images/36632.jpg" width="100%" height="100%"/>
+  </div>
+  <div class="img-container">
+    <img src="images/49124.jpg" width="100%" height="100%"/>
+	</div>
+	<div class="img-container">
+		<img src="images/729999.jpg" width="100%" height="100%"/>
+	</div>
+	<div class="img-container">
+		<img src="images/730178.jpg" width="100%" height="100%"/>
+	</div>
+	<div class="img-container">
+		<img src="images/730182.jpg" width="100%" height="100%"/>
+	</div>
+	<div class="img-container">
+		<img src="images/730259.jpg" width="100%" height="100%"/>
+	</div>
+	<div class="img-container">
+		<img src="images/flower.jpg" width="100%" height="100%"/>
+	</div>
+	<div class="img-container">
+		<img src="images/wallhaven-535670.jpg" width="100%" height="100%"/>
+	</div>
+</div>
+`
+
+export default _galleryContent;
```

Comment:
Let's add an empty line at the end of your files.

Reply:
Okay